src/actinia_api/swagger2/actinia_core/apidocs/resource_management.py

```diff
@@ -70,7 +71,16 @@
 
 resource_put_doc = {
     "tags": ["Resource Management"],
-    "description": "Updates/Resumes the status of a resource. "
+    "description": "Updates/Resumes the status of a failed resource. This "
+    "assumes that 'save_interim_results' is configured. "
+    "The job will restart at the step before the failed one and "
+    "execute these steps in the possibly corrected process chain sent along "
+    "with it. If 'save_interim_results' is set to 'True' the temporary mapset"
+    " is saved before the failed step and the resumption should work without "
+    "any problem. But if 'save_interim_results' is set to 'onError' the "
+    "temporary mapset is only saved if an error occurs and so the mapset "
+    "may contain changes from the failed step what may make resumption "
+    "difficult or impossible."
```

Comment:
Maybe suggest solutions in this case? And also add to [job_resumption.md file in actinia-core MR](https://github.com/mundialis/actinia_core/pull/390/files)